accounts/serializers.py

```diff
@@ -106,10 +103,14 @@ class Meta:
             'email_new_studies',
             'email_study_updates',
             'email_response_questions',
+            'date_created',
         )
 
-    # class JSONAPIMeta:
-    #     included_resources = ['demographics', ]
+class FullUserSerializer(BasicUserSerializer):
+    
+    class Meta():
+        model = User
+        fields = BasicUserSerializer.Meta().fields + ('username',)
```

Comment:
Also, consider adding tests for this. We don't have good test coverage in this django app, but we do have API tests.  A test to make sure that usernames only show if you have this permission, and do not show if you don't, would be useful because of the sensitivity of this field.

Reply:
I have added some basic tests - not exhaustive but just to check we see the username field with this permission but not if user is just admin with can_read_all_user_data.